app/controllers/application_controller.rb

```diff
@@ -151,4 +151,9 @@ def pagination_params
   def render_job_id(jid)
     render json: { job_id: jid }, status: 202
   end
+
+  def append_info_to_payload(payload)
```

Comment:
append_info_to_payload is part of the rails magic in rails/actionpack/lib/action_controller/metal/instrumentation.rb

Reply:
👍nice find